ArduCopter/GCS_Mavlink.cpp

```diff
@@ -993,6 +993,36 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_MANUAL_CONTROL:
+    {
+        if(msg->sysid != copter.g.sysid_my_gcs) break;                         // Only accept control from our gcs
```

Comment:
I don't think that's needed for a such a simple piece of code. Also it follows what is done above in the RC overrides message.